File: python/models/persona.py
class Persona:
    def __init__(self, cedula, nombre_completo, correo):
        self.cedula = cedula
        self.nombre_completo = nombre_completo
        self.correo = correo

    def mostrar_datos(self):
        return (
            f"Cédula: {self.cedula}\n"
            f"Nombre: {self.nombre_completo}\n"
            f"Correo: {self.correo}"
        )

File: java/Persona.java
public abstract class Persona {
    private final String cedula;
    private final String nombre;
    private final String correo;

    protected Persona(String cedula, String nombre, String correo) {
        this.cedula = cedula;
        this.nombre = nombre;
        this.correo = correo;
    }

    public String getCedula() {
        return cedula;
    }

    public String getNombre() {
        return nombre;
    }

    public String getCorreo() {
        return correo;
    }

    public String mostrarDatos() {
        return cedula + " | " + nombre + " | " + correo;
    }
}
